CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - **[FIX]** Fix markup `Item` tag label by replacing `div` children with `span`
 - **[BREAKING CHANGE]** Import `icons` as default instead of named export for a better code splitting
 - **[UPDATE]** Make `Review` more semantic with proper HTML and Review/Rating microdata
+- **[UPDATE]** Add a11y unit test for `ItemCheckbox`
```

Comment:
Not necessary IMHO to add this in the changelog. No external impact

Reply:
Yep done